examples/subscribe_notify_characteristic.rs

```diff
@@ -0,0 +1,118 @@
+use std::time::Duration;
+
+use btleplug::api::{ValueNotification, CharPropFlags};
+use btleplug::platform::{Adapter, Manager};
+use btleplug::api::{Central, Manager as _, Peripheral};
+use std::io::Cursor;
+use std::error::Error;
+use uuid::Uuid;
+use tokio::time;
+use futures::stream::{StreamExt};
+
+const PERIPHERAL_NAME_MATCH_FILTER: &'static str = "Neuro"; // filter BLE device by partial name
+const NOTIFY_CHARACTERISTIC_MATCH_UUID: &'static str = "6E400002-B534-F393-67A9-E50E24DCCA9E"; // subscribe UUID
+
+#[cfg(target_os = "linux")]
+fn print_adapter_info(adapter: &Adapter) {
+    println!("connected adapter {:?}", adapter);
+}
+
+#[cfg(any(target_os = "windows", target_os = "macos"))]
```

Comment:
This platform-specific stuff shouldn't be necessary anymore.

Reply:
ok, I will remove it